features/visitor_can_view_listing.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Visitor can view listing
+  As a Visitor
+  In order to find a suitable host for my cat
+  I want to be able to view listings on the landing page
+
+  Background:
+    Given the following listings exist
+    | name | location   | description | email        | phone_number | availability |
+    | Zane | Gothenburg | I'm nice    | zane@cat.com | 0701234567   | 1-10 may     |
```

Comment:
This is supposed to be an index of several listings, please add more listings

Reply:
will do